src/doc/rustdoc/src/unstable-features.md

```diff
@@ -310,6 +310,8 @@ the source.
 
 ### `--show-type-layout`: add a section to each type's docs describing its memory layout
 
+* Trackin issue: [#113248](https://github.com/rust-lang/rust/issues/113248)
```

Comment:
```suggestion
* Tracking issue: [#113248](https://github.com/rust-lang/rust/issues/113248)
```

Reply:
Corrected to say `Trackin' issue` instead of `Trackin issue`

(just kidding, should be resolved)